docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3'
```

Comment:
We should place this `docker-compose.yaml` only in one place, either `compendium` or `compendium-example`, and refer it from the other one.
IMHO, I think we should refer the `docker-compose.yaml` file in the README of this project, since it will be easier to keep up to date in `compendium`.

Reply:
that makes sense to me